### ISSUE TITLE
Need help with the output

### ISSUE THREAD

**Issue by Abhinav-235** [Genymobile/scrcpy#5277]
the video lag and the output fps is like 10. I am pretty new to scrcpy and i dont know how to run commands. please need a full walkthrough. when i am in the lobby of the game BGMI (India) the output seems fine but as soon as I start the game the output goes like in the video. Hope anyone knows how to deal with it!!!!!!!

https://github.com/user-attachments/assets/6a01616d-06fc-44ac-bf39-eba2f6a60ec6

the game eventuallly came to an hault

**Comment by rom1v** [comment]
Your device hardware is not able to render the game and encode at the same time with good performances.

Try reducing the capture resolution: `scrcpy -m1024`.

**Comment by rom1v** [comment]
> i dont know how to run commands.

https://github.com/Genymobile/scrcpy/blob/master/doc/windows.md#run

**Comment by Abhinav-235** [comment]
yeah the problem still presumes, until i enter the game the output is very good as soon as the game starts it goes into slow mo

**Comment by Abhinav-235** [comment]
I am using a Poco X6 pro the ram is 8gb, does the laptop specs will affect this?

**Comment by Abhinav-235** [comment]
the audio doesnt lag only the video output is like this!

**Comment by rom1v** [comment]
> I am using a Poco X6 pro the ram is 8gb, does the laptop specs will affect this?

No, it's the device. You can confirm by recording: `scrcpy --record=file.mp4 -m1024`.
If you play the file on the computer, it will lag (because it is captured laggy).

Also `scrcpy --print-fps` will log the capture frame rate to the console.

> the audio doesnt lag only the video output is like this!

It's the video capture, which cannot be performed efficiently while the game is running (your hardware probably share some resources between game rendering and video encoding).

**Comment by Abhinav-235** [comment]
so there is no solution for this, i gotta go for paid apps??

**Comment by rom1v** [comment]
> so there is no solution for this, i gotta go for paid apps??

I have no solution since it's a limitation of the device hardware. I guess you'll get the same problem with paid apps (but I might be wrong, please tell us).

**Comment by chacalau** [comment]
you can try using adb command to make the device screen size smaller

`adb -s DEVICEID shell wm size 800x1200`

also try set the max-size smaller in scrcpy command

`--max-size=1024`
